changelog/unreleased/enterprise-issue-11092.toml

```
@@ -0,0 +1,14 @@
+type = "a"
+message = "Lookup table page UI revamp"
+
+issues = [
+  "Graylog2/graylog-plugin-enterprise#11108",
+  "Graylog2/graylog-plugin-enterprise#11107",
+  "Graylog2/graylog-plugin-enterprise#11115",
+  "Graylog2/graylog-plugin-enterprise#11110",
+  "Graylog2/graylog-plugin-enterprise#11111",
+  "Graylog2/graylog-plugin-enterprise#11109",
+  "Graylog2/graylog-plugin-enterprise#11113",
+]
```

You don't need to include all of the task issues that we make in the team in changelogs. The only issue the changelog needs to include are bug/feature level issues. In this case, 11092 should be fine.
```suggestion
issues = ["Graylog2/graylog-plugin-enterprise#11092"]
```